lib/device.js

```diff
@@ -285,7 +267,7 @@ const util = require('util'),
                     captureFormat = "PNG"; // PNG is default format
 
                 if (options.outputPath === null){
-                    captureFileName = _createDateFileName();
+                    captureFileName = options.session.target.name + "_" + "display" + options.display + "_" + createDateFileName("") + ".png";
```

Comment:
Why don't you modify this? 
createDateFileName("", ".png")? 
And make sure what is the first parameter or add Null? 
Let's have a talk.